statsd/example/statsd_main.go

```diff
@@ -21,21 +21,26 @@
 package main
 
 import (
+	"io"
 	"log"
 	"math/rand"
 	"time"
 
+	"github.com/cactus/go-statsd-client/statsd"
 	"github.com/uber-go/tally"
 	statsdreporter "github.com/uber-go/tally/statsd"
-
-	"github.com/cactus/go-statsd-client/statsd"
 )
 
 // To view statsd emitted metrics locally you can use
 // netcat with "nc 8125 -l -u"
 func main() {
-	statter, err := statsd.NewBufferedClient("127.0.0.1:8125",
```

Comment:
This is [deprecated](https://github.com/cactus/go-statsd-client/blob/7eb0950793aa9a679bc468eb1da9b5520f73b16f/statsd/client_legacy.go#L32) now